File: dart/test/seed_audio_ai_site_kit_test.dart
import 'package:test/test.dart';
import 'package:seed_audio_ai_site_kit/seed_audio_ai_site_kit.dart';

void main() {
  test('metadata is available', () {
    expect(seedAudioAiSite['name'], equals('Seed Audio AI'));
    expect(seedAudioAiSite['homepage'], equals('https://seedaud.io/'));
  });

  test('URLs are normalized', () {
    expect(seedAudioUrl(), equals('https://seedaud.io/'));
    expect(seedAudioUrl('/pricing/'), equals('https://seedaud.io/pricing/'));
  });
}

File: elixir/test/seed_audio_ai_site_kit_test.exs
defmodule SeedAudioAISiteKitTest do
  use ExUnit.Case

  test "metadata is available" do
    assert SeedAudioAISiteKit.site().name == "Seed Audio AI"
    assert SeedAudioAISiteKit.site().homepage == "https://seedaud.io/"
  end

  test "URLs are normalized" do
    assert SeedAudioAISiteKit.seed_audio_url() == "https://seedaud.io/"
    assert SeedAudioAISiteKit.seed_audio_url("/pricing/") == "https://seedaud.io/pricing/"
  end
end
